src/electionguard-ui/ElectionGuard.UI/ViewModels/TallyProcessViewModel.cs

```diff
@@ -137,19 +169,53 @@ private async Task RejectTally()
     [RelayCommand(CanExecute = nameof(CanStartTally))]
     private async Task StartTally()
     {
-        _ = await Task.FromResult(() =>
+        if (Tally is null)
         {
-            if (!(_timer?.IsRunning ?? true))
-            {
-                _timer?.Start();
-            }
+            return;
+        }

```

Comment:
if check being done in the CanExecute.  Not needed here.  Could just use ! on the Tally below.